src/test/lazy-omap-stats/lazy_omap_stats_test.cc

```diff
@@ -563,10 +563,7 @@ void LazyOmapStatsTest::wait_for_active_clean()
   cout << "Waiting for active+clean" << endl;
 
   int index = -1;
-  regex reg(
-      "\n"
-      R"((PG_STAT[\s\S]*))"
-      "\n +\n[0-9]");
+  regex reg(R"(PG_STAT[^\n]*\n)");
   string command = R"({"prefix": "pg dump"})";
```

Comment:
Of course these are not equivalent and the change should not be necessary but, if you are confident this covers all bases then I'm all for it.